cylc/flow/parsec/config.py

```diff
@@ -191,18 +184,26 @@ class ConfigNode(ContextNode):
         display_name (str):
             This is the user-facing name of the config.
             Note the regular ``name`` might be ``__MANY__``.
+        meta (ConfigNode):
+            Another ConfigNode to use as a template for this one.
+
+            This is useful if you want to create a specific instance of
+            a generic configuration e.g. ``[elephant]`` from ``[<animal>]``.
+
+            Leaf nodes inherited from the generic config wil have
+            ``meta=True``.
 
     """
 
-    ROOT_NAME_FMT = '{display_name}:'
+    ROOT_NAME_FMT = '{display_name}'
     NODE_NAME_FMT = '[{display_name}]'
     LEAF_NAME_FMT = '{display_name}'
```

Comment:
The `rear` for `ConfigNode`s used to be `file:[section]setting`.

This changes it to `file[section]setting` and `file|setting` to match `cylc-sphinx-extensions` (so the `repo` of a node is now the same as the Sphinx reference to a node).